models/image.go

```diff
@@ -0,0 +1,28 @@
+package models
+
+import (
+	"time"
+
+	"github.com/zpatrick/go-plugin-swagger"
+)
+
+type Image struct {
```

Comment:
:-1: I'm not a fan of this model and api schema - it doesn't mesh well with what currently exists and could be made much more simple/intuitive. 

 I think the total scheme should look something like this (listing things that would need to be changed):

```
GET /repo/{owner}/{name}
// no more "image_tags" field
{
  "owner": "xfra",
  "name": "l0-api"
}
```

```
GET /repo/{owner}/{name}/image
{
  "images": {
    "v0.1.0": {
      "digest": "xxyyzzz",
      "size": "10MB",
      "timestamp": "1111-22-33 44:55"
    },
    "latest": {
      "digest": "xxyyzzz",
      "size": "10MB",
      "timestamp": "1111-22-33 44:55"
    }
  }
}
```


```
GET /repo/{owner}/{name}/image/{tag}
{
  "digest": "xxyyzzz",
  "size": "10MB",
  "timestamp": "1111-22-33 44:55"
}
```

As an aside - the `/images` endpoint doesn't need to exist. It only existed in the old d.ims.io because we didn't have an API for it.